cloudlaunchserver/values.yaml

```diff
@@ -97,7 +97,11 @@ postgresql:
   enabled: true
   postgresqlDatabase: cloudlaunch
   postgresqlUsername: cloudlaunch
-  postgresqlPassword: some_pass
+  # Password will be auto-generated if not specified, then stored in a secret
+  # postgresqlPassword: some_pass
+  # The postgres superuser password will not be set if not specified. If you need a postgres superuser,
+  # this should be set at launch (eg: for access to manually modify databases on the running server)
+  # postgresqlPostgresPassword: admin_pass
```

Comment:
Is it possible for us to autogenerate a superuser password as well?

Reply:
I would say maybe we do it in cloudman-boot if we want to do it. Not sure if there is an easy way to do it through values unless we keep the default `postgres` username then add `cloudman/cloudlaunch` instead of our current approach to create the user through `postgresqlUsername`

Reply:
That does sound sensible to me. I think renaming the postgres superuser was my mistake, and creating a separate cloudlaunch user seems like the right way to do this.